partner-portal/src/UI/package.json

```diff
@@ -8,8 +8,7 @@
     "watch": "ng build --watch --configuration development",
     "test": "ng test",
     "prestart": "node aspnetcore-https",
-    "start:windows": "ng serve --ssl --ssl-cert \"%APPDATA%\\ASP.NET\\https\\%npm_package_name%.pem\" --ssl-key \"%APPDATA%\\ASP.NET\\https\\%npm_package_name%.key\" --host=127.0.0.1",
-    "start:default": "ng serve --ssl --ssl-cert \"$HOME/.aspnet/https/${npm_package_name}.pem\" --ssl-key \"$HOME/.aspnet/https/${npm_package_name}.key\" --host=127.0.0.1"
+    "gen-api": "ng-openapi-gen -i http://localhost:8080/swagger/v1/swagger.json -o ./src/app/shared/api --ignoreUnusedModels true"
```

Comment:
we are also using "gen:api", which one is correct?

Reply:
We can use gen:api or gen-api . This has to be followed what was written in the package.Json for generating the API. I will make them consistent but need change on other portals